### ISSUE TITLE
learning rate and decay?

### ISSUE THREAD

**Issue by Rock-L** [macanv/BERT-BiLSTM-CRF-NER#83]
hi macan:

when i training the ner model , the loss from 44 to 9.2, but loss decrease very slowly , i think the learning rate decay so small . but i have no way to trace the learning rate , and decay can not be found. do you have any solutions ?

**Comment by macanv** [comment]
you can add training hook when you training,
just like in line 411 to 414 in bert_base/train/bert_lstm_crf.py
About more infomation of hook, you may go to read tensorflow api document.

**Comment by Rock-L** [comment]
i have tried this way ,
![image](https://user-images.githubusercontent.com/22190647/54012966-a1708200-41b2-11e9-9999-8226c3333604.png)
but get an error like :
![image](https://user-images.githubusercontent.com/22190647/54013207-3ffce300-41b3-11e9-8d5a-5de6a632eb9a.png)

**Comment by xmy7216** [comment]
> i have tried this way ,
> ![image](https://user-images.githubusercontent.com/22190647/54012966-a1708200-41b2-11e9-9999-8226c3333604.png)
> but get an error like :
> ![image](https://user-images.githubusercontent.com/22190647/54013207-3ffce300-41b3-11e9-8d5a-5de6a632eb9a.png)

Hi, I have the same problem. How do you solve this problem in the end?  Thank you very much.
